src/components/FilterGroup.tsx

```diff
@@ -67,19 +74,23 @@ export function FilterGroup({
 }: PropsWithChildren<FilterGroupProps>) {
   const cssClasses = useMemo(() => {
     const { option, optionLabel, optionInput, ...remainingClasses } = customCssClasses;
-    return {
+    return useComposedCssClasses(builtInCssClasses, {
```

Comment:
linter issue: React Hook "useComposedCssClasses" cannot be called inside a callback